src/Simplex/Messaging/Notifications/Types.hs

```diff
@@ -99,8 +99,8 @@ type NtfActionTs = UTCTime
 data NtfSubNTFAction
   = NSACreate
   | NSACheck
-  | NSADelete
-  | NSARotate
+  | NSADelete -- deprecated
+  | NSARotate -- deprecated
```

Comment:
why deprecated?

Reply:
as in no longer used - only keeping them for processing legacy records that may have them saved as action